### ISSUE TITLE
Make doctor check environment variables

### ISSUE THREAD

**Issue by RadicalZephyr** [bike-barn/hermit#16]
For instance it should inspect `HERMIT_ROOT` and `HERMIT_PROFILE` for being valid directories.

**Comment by RadicalZephyr** [comment]
Issue #16 should also be part of what `doctor` does.

**Comment by reillysiemens** [comment]
How does `hermit doctor` check that you have a properly updated hermit? Get the most recent git tags and see if you're behind?

**Comment by RadicalZephyr** [comment]
yup.  Won't work currently cause I don't think there are any tags, but seems like a sound idea

**Comment by reillysiemens** [comment]
I think it's reasonable to do a pre-release tag. I think there's a special way to denote that so that GitHub knows it's not a full release.

**Comment by RadicalZephyr** [comment]
I added a pre-release tag, so now this has something to target.

**Comment by RadicalZephyr** [comment]
#36 should also be part of `doctor`
